larray/inout/array.py

```diff
@@ -505,7 +505,7 @@ def read_excel(filepath, sheet=0, nb_axes=None, index_col=None, fill_value=np.na
     Parameters
     ----------
     filepath : str
-        Path where the Excel file has to be read or use -1 to refer to the last active workbook.
+        Path where the Excel file has to be read or use -1 to refer to the currently active workbook.
```

Comment:
I just noticed open_excel uses the same wording (and possibly LArray.to_excel too), so if we change this, we probably need to change the other(s) too. Unsure it's worth it though.